### ISSUE TITLE
Background reading

### ISSUE THREAD

**Issue by Hilary-Simpson** [fac-14/sleuth-coop#1]

**Comment by thejoefriel** [comment]
@Hilary-Simpson Do you mind confirming you have sent the relevant background content to Dominic please?

**Comment by Hilary-Simpson** [comment]
This is our website with the marketing material. And the branding

**Comment by Hilary-Simpson** [comment]
Egress is a masterclass for SMEs marketing their wares to Local Government 
https://www.sleuth.coop/our-research

**Comment by developess** [comment]
Companies: Semarchy, Entity, Bae Systems, Senzing, Infordata, Winpure, Stibo, Orchestra, Talend, Riversand, Stambia,

**Comment by Hilary-Simpson** [comment]
> Entity Resolution / Data matching / Single view tech Companies: Semarchy, Entity, Bae Systems, Senzing, Infordata, Winpure, Stibo, Orchestra, Talend, Riversand, Stambia,

And Xantura

**Comment by Hilary-Simpson** [comment]
This is the technology that I want to be able to explain in plain English to Local gov business people! 
https://medium.com/@jeffjonas/why-federated-search-bites-for-enterprise-discovery-2235ca9b5b10
